t/apicast-subset-of-services.t

```diff
@@ -1,47 +1,48 @@
-use lib 't';
-use Test::APIcast 'no_plan';
+use Test::APIcast::Blackbox 'no_plan';
 
 run_tests();
 
 __DATA__
 
 === TEST 1: multi service configuration limited to specific service
---- main_config
-env APICAST_SERVICES_LIST=42,21;
---- http_config
-  include $TEST_NGINX_UPSTREAM_CONFIG;
-  lua_package_path "$TEST_NGINX_LUA_PATH";
-  init_by_lua_block {
-    require('apicast.configuration_loader').mock({
-      services = {
-        {
-          id = 42,
-          backend_version = 1,
-          proxy = {
-            api_backend = "http://127.0.0.1:$TEST_NGINX_SERVER_PORT/api-backend/one/",
-            hosts = { 'one' },
-            proxy_rules = {
-              { pattern = '/', http_method = 'GET', metric_system_name = 'one', delta = 1 }
-            }
+--- env eval
+("APICAST_SERVICES_LIST", "42,21")
+--- configuration
+{
+  "services": [
+    {
+      "backend_version": 1,
+      "proxy": {
+        "hosts": [
+          "one"
+        ],
+        "api_backend": "http://test:$TEST_NGINX_SERVER_PORT/api-backend/one/",
```

Comment:
I know it was part of the original test file, but maybe we can take this opportunity to remove things that do not add any value.

The `/api-backend/one` part looks unnecessary for this test. If we remove it, we'll be able to also delete some things in the `upstream`block below (`~ /api-backend(/.+)` and the usage of `$1`).

Reply:
Same for TEST 2